scripts/generator/graphql_generator/csharp/type.cs.erb

```diff
@@ -120,14 +120,19 @@ namespace <%= namespace %>.GraphQL {
     <% when 'INPUT_OBJECT' %>
         <%= docs_input_object(type) %>
         public class <%= type.classify_name %> : InputBase {
+
+            <% type.input_fields.each do |field| %>
+            public const string <%= field.name %>FieldKey = "<%= field.name %>";
```

Comment:
This is public because some later PR is going to need to reference these (or could these be made private, seems like, at least in this PR alone, they are used only internally to the given class)?